src/OpenTelemetry.Instrumentation.GrpcCore/OpenTelemetry.Instrumentation.GrpcCore.csproj

```diff
@@ -1,10 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net6.0;netstandard2.1;netstandard2.0</TargetFrameworks>
```

Comment:
What is the reason for adding new TFMs?

What is more, tests are executed only against net6.0. NET Standard 2.1/2.0 will be not covered.

Reply:
To use nullable annotated BCL and use nullable attribute `[NotNull]` in unit-tests (currently there is the conflict with the one defined in `Guard.cs`), but i agree it is not the best idea due to compatibility with not annotated netstandard2.0 BCL and the point you mentioned. Reverted this change and replaced `[NotNull]` attribute with `Assert.NotNull` checks.